common/test/acceptance/pages/lms/courseware.py

```diff
@@ -118,6 +118,16 @@ def start_timed_exam(self):
         self.q(css=".xblock-student_view .timed-exam .start-timed-exam").first.click()
         self.wait_for_element_presence(".proctored_exam_status .exam-timer", "Timer bar")
 
+    def stop_timed_exam(self):
+        """
+        clicks the start this timed exam link
```

Comment:
I see student (LMS) acceptance tests for Timed Exams in common/tests/acceptance/tests/lms/test_lms_instructor_dashboard.py (which is probably not the best place for them). I think because we built off of acceptances tests written for instructor dashboard. Let me see if I can refactor a bit.